contracts/merger/PegExchangerDripper.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.8.4;
+
+import "./PegExchanger.sol";
+
+contract PegExchangerDripper {
+
+    IERC20 public constant TRIBE = IERC20(0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B);
+    address public constant CORE = 0x8d5ED43dCa8C2F7dFB20CF7b53CC7E593635d7b9;
+    address public constant PEG_EXCHANGER = 0xc09BB5ECf865e6f69Fe62A43c27f036A426909f7;
+    
+    uint256 public constant THRESHOLD = 5e18; // 5M TRIBE cutoff for dripping
+    uint256 public constant DRIP_AMOUNT = 20e18; // drip 20M TRIBE
```

Comment:
this will only drip 20 tribe.

bump up to 20_000_000e18

Reply:
done